spring-batch-docs/asciidoc/scalability.adoc

```diff
@@ -299,17 +302,17 @@ Spring Batch does however provide a useful implementation of
 </bean>
 ----
 
-The `gridSize` determines the number of separate
+The `gridSize` attribute determines the number of separate
       step executions to create, so it can be matched to the size of the
-      thread pool in the `TaskExecutor`, or else it can
-      be set to be larger than the number of threads available, in which case
-      the blocks of work are smaller.
+      thread pool in the `TaskExecutor`. Alternatively, it can
+      be set to be larger than the number of threads available, which makes
+      the blocks of work smaller.
 
-The `TaskExecutorPartitionHandler` is quite
-      useful for IO intensive `Steps`, like copying large numbers of files or
+The `TaskExecutorPartitionHandler` is
+      useful for IO-intensive `Step` implementations, such as copying large numbers of files or
```

Comment:
`instances` instead of `implementations`

Reply:
Yes, please. Thank you very much for the review and the commit work, Glenn. :)